src/test/kotlin/three/consulting/epoc/controller/CustomerControllerIntegrationTest.kt

```diff
@@ -12,4 +13,19 @@ class CustomerControllerIntegrationTest : ControllerIntegrationTest() {
         val response = restTemplate.getForEntity("/customer/1", ObjectNode::class.java)
         assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
     }
+
+    @Test
+    fun `add customer returns 200`() {
+        val httpEntity = jsonPostEntity("src/test/resources/customer/validCreation.json"
+        )
+        val response = restTemplate.postForEntity("/customer", httpEntity, ObjectNode::class.java)
+        assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+    }
+    @Test
+    fun `add customer with id returns 400`() {
```

Comment:
You're not adding a customer with id, but with a missing name here. Also typo in the json file naming unvalid -> invalid